Sources/SafeDICore/Models/ConcreteDeclSyntaxProtocol.swift

```diff
@@ -55,6 +55,19 @@ extension ClassDeclSyntax: ConcreteDeclSyntaxProtocol {
     public var declType: ConcreteDeclType { .classType }
 }
 
+extension EnumDeclSyntax: ConcreteDeclSyntaxProtocol {
+    public var keyword: SwiftSyntax.TokenSyntax {
+        get {
+            enumKeyword
+        }
+        set {
+            enumKeyword = newValue
+        }
+    }
+
+    public var declType: ConcreteDeclType { .enumType }
+}
```

Comment:
We now enable visiting `EnumDeclSyntax` types, because `@Instantiable` types may be declared within them.